cves/CVE-2023-33170.yaml

```diff
@@ -0,0 +1,11 @@
+affectedPackages:
+  - cpe23Uri: cpe:2.3:a:microsoft:microsoft.netcore.app:*:*:*:*:*:*:*:*
+    versionEndExcluding: 6.0.20
+    versionStartIncluding: 6.0.0
+    vulnerable: true
+  - cpe23Uri: cpe:2.3:a:microsoft:microsoft.netcore.app:*:*:*:*:*:*:*:*
+    versionEndExcluding: 7.0.9
+    versionStartIncluding: 7.0.0
+    vulnerable: true
+id: CVE-2023-33170
```

Comment:
LGTM, but Is it not CVE-2023-33127?

Reply:
It is CVE-2024-33170. I copied the wrong CVE in the title and commit.